app/models/mixins/authentication_mixin.rb

```diff
@@ -288,7 +288,6 @@ def authentication_check_no_validation(type, options)
           [:error, err]
         end
       end
-
```

Comment:
pure whitespace..

And I'm wondering whether `ems_middleware/index.html.haml` is really needed since it's empty..?